### DISCUSSION TITLE
Why is the phoenix data processed by resistics library completely different from the results obtained by ssmt2000

### DISCUSSION THREAD

**Discussion by mshe0528** [resistics/resistics#43] [Q&A]
When I read the phoenix data, there was an error. When I separated the ts3,ts3 and ts5 files, it meant that they were divided into three folders, and each folder had tbl header file and ts file. In this way, the code only calibrated the data once when calibrating the file. CLB and CLC need to be calibrated twice, I added a piece of code to make it calibrate twice, but the result is completely different from the result processed on ssmt2000, why is this? I really can't find the problem, please help answer it.
        If you need any data, please contact me at the following email address: 2791953613@qq.com

**Comment by Morphlinggod** [comment]
如果你把v8格式的数据先转换到其他格式的数据，然后再放入这个项目中进行解析，结果会不会不一样呢？
